docs/sphinx/source/api.rst

```diff
@@ -97,6 +97,7 @@ Filtering
    :toctree: generated/
     
    clip_filter
+   geometric_clip_filter
```

Comment:
Update with the other functions too

Reply:
Addressed in commit 390762799ae1e298fc56be5c778936c05426941d